server/views/azuread-b2c-login.html

```diff
@@ -39,23 +39,24 @@
       }
       .govuk-error-summary {
         margin-top: 25px;
-				margin-bottom: 10px;
+        margin-bottom: 10px;
       }
-			.darts-error {
+      .darts-error {
         border-left: 5px solid #D4351C;
         padding-left: 15px;
       }
-			.darts-error input {
-				border-color: #D4351C !important;
-				border-width: 2px !important;
-			}
-			.error.itemLevel {
-				font-weight: bold;
-				margin: 0;
-			}
-			.error.itemLevel p {
-				margin: 10px 0 15px 0;
-			}
+      .darts-error input {
+        border-color: #D4351C !important;
+        border-width: 2px !important;
+      }
+      .error.itemLevel {
+        font-weight: bold;
+        margin: 0;
+      }
+      .error.itemLevel p {
+        margin: 10px 0 15px 0;
+        color: #D4351C;
```

Comment:
The addition of this line is the only change here. Otherwise I formatted the indents to use spaces rather than tabs.